packages/common/src/chains/mainnet.json

```diff
@@ -80,15 +80,20 @@
       "block": 12965000, 
       "forkHash": "0xb715077d"
     },
+    {
+      "name": "arrowGlacier",
+      "block": 13773000,
+      "forkHash": null
```

Comment:
Fork hash still missing here, can be taken from https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/arrow-glacier.md